operator/v1/types_olm.go

```diff
@@ -13,6 +13,7 @@ import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:path=olms,scope=Cluster
 // +kubebuilder:subresource:status
+// +kubebuilder:metadata:annotations=include.release.openshift.io/ibm-cloud-managed=false
```

Comment:
Lets try adding the partner annotation and see if that brings the generation back 
```suggestion
// +kubebuilder:metadata:annotations=include.release.openshift.io/ibm-cloud-managed=false
// +kubebuilder:metadata:annotations=include.release.openshift.io/self-managed-high-availability: "true"
```

Reply:
// +kubebuilder:metadata:annotations=include.release.openshift.io/self-managed-high-availability=true